umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/collection_progress.clj

```diff
@@ -0,0 +1,17 @@
+(ns cmr.umm-spec.xml-to-umm-mappings.collection-progress
+  "Functions for parsing UMM collection progress records out of XML documents of other formats."
+  (:require
+   [clojure.string :as string]
+   [cmr.common.xml.parse :refer :all]
+   [cmr.common.xml.simple-xpath :refer :all]
+   [cmr.umm-spec.util :as umm-spec-util]))
+
+(defn get-collection-progress
```

Comment:
I like this. You can make it even more generic. It does not have to be just for collection progress, it can be for any enum. That way @mschmele could use it and it could be used for the upcoming enum tickets.